Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -85,6 +86,8 @@ void StackingContext::paint(PaintContext& context)
     // Draw the non-positioned floats (step 5)
     paint_descendants(context, m_box, StackingContextPaintPhase::Floats);
     // Draw inline content, replaced content, etc. (steps 6, 7)
+    if (is<ReplacedBox>(m_box))
+        m_box.paint(context, PaintPhase::Foreground);
```

Comment:
We don't need to check for `ReplacedBox` here, we should draw foregrounds for every box that has it (that also fixes acid3)